plugins/basic/python/tk_unreal_basic/plugin_bootstrap.py

```diff
@@ -87,7 +90,7 @@ def _initialize_manager(plugin_root_path):
 
     # open the yaml file and read the data
     with open(plugin_info_yml, "r") as plugin_info_fh:
-        plugin_info = yaml.load(plugin_info_fh)
+        plugin_info = yaml.load(plugin_info_fh, yaml.SafeLoader)
```

Comment:
I added this to get rid of a yaml warning, that was not in the first PR you reviewed.